Sources/Orbit/Components/InputField.swift

```diff
@@ -160,15 +215,45 @@ public extension InputField {
     }
 }
 
+// MARK: - Model
+private class SecureViewModel: ObservableObject {
+
+    @Published var isShowSecuredText = false {
```

Comment:
"isShow"?

As I don't know exactly what is meant here, some options that come to mind are `isShowingSecuredText`, `showSecureText`, `showsSecureText`...